spec/integration/outputs/index_spec.rb

```diff
@@ -0,0 +1,129 @@
+# encoding: utf-8
+require "logstash/devutils/rspec/spec_helper"
+require "logstash/outputs/elastic_app_search"
+require "logstash/codecs/plain"
+require "logstash/event"
+require "json"
+
+describe "indexing against running AppSearch", :integration => true do
+
+  let(:engine_name) {
+    (0...10).map { ('a'..'z').to_a[rand(26)] }.join
+  }
+
+  let(:config) {
+    {
+      "api_key" => ENV['APPSEARCH_PRIVATE_KEY'],
+      "engine" => engine_name,
+      "url" => "http://appsearch:3002"
+    }
+  }
```

Comment:
Nitpick: multiline `do`...`end`

```suggestion
  let(:config) do
    {
      "api_key" => ENV['APPSEARCH_PRIVATE_KEY'],
      "engine" => engine_name,
      "url" => "http://appsearch:3002"
    }
  end
```